### ISSUE TITLE
Better focuser feedback

### ISSUE THREAD

**Issue by JunCTionS** [GLORIA-project/solar-experiment-interface#8]
Related to bugs #1 and #4 but on a normal user level. Yesterday I worked with high-school students group and we were unable to control the focuser.

We attempted to change the focus, and didn't find it responsive until eventually the focus was completely lost, and we were unable to return it to the original value. 

I have no idea if this is due to a slow response from this specific telescope, but not knowing what the current state of the focuser was (which was obviously different than the one we set) gave us very little impression of control.

Also the slider is very difficult to be set to a specific number of steps, as we were trying to get it back to 0 to attempt the original focus hitting + or - sent it several steps above or below the desired target. At any case we didn't know if this mattered much due to the slow (or erratic) response of the focusing mechanism.

In summary, I request a numerical feedback of the actual position of the focuser (not what the system is telling it to move to, but what the focuser says it's currently at) and also a way to easily move single steps.
